README.md

```diff
@@ -9,84 +9,147 @@ A [Test Kitchen][kitchenci] Driver for Amazon EC2.
 This driver uses the [aws sdk gem][aws_sdk_gem] to provision and destroy EC2
 instances. Use Amazon's cloud for your infrastructure testing!
 
+## Initial Setup
+
+To get started, you need to install the software and set up your credentials and SSH key. Some of these steps you have probably already done, but we include them here for completeness.
+
+1. Install the latest ChefDK and put it in your path.
+2. From this repository, type `bundle install; bundle exec rake install` to install the latest version of the driver.
```

Comment:
The instructions in this readme cover the code in `master`--the things it says are not true of earlier versions. I think it'd be best to hold off and change it to `gem install` after we release the code. Once we check this in, we can release an alpha or something, and point at that ...